processor/k8sattributesprocessor/internal/kube/client_test.go

```diff
@@ -464,7 +464,7 @@ func TestExtractionRules(t *testing.T) {
 				{
 					APIVersion: "batch/v1",
 					Kind:       "Job",
-					Name:       "pi",
+					Name:       "auth-cronjob-27667920-t9hsb",
```

Comment:
This doesn't seem like a job name, it's rather pod name, job would be `auth-cronjob-27667920`